htdocs/admin/prelevement.php

```diff
@@ -91,6 +92,12 @@
         if (! $res > 0) $error++;
     }
 
+    if (GETPOST("PRELEVEMENT_ADDDAYS") || GETPOST("PRELEVEMENT_ADDDAYS")=="")
+    {
+        $res = dolibarr_set_const($db, "ADDDAYS", GETPOST("PRELEVEMENT_ADDDAYS"),'chaine',0,'',$conf->entity);
```

Comment:
Can you rename the constant ADDDAYS into
PRELEVEMENT_ADDDAYS
Each constant must be name with MODULENAME_NAMEOFCONSTANT

Reply:
Ok. I hope, I cleaned everything up right